dev/sparktestsupport/modules.py

```diff
@@ -436,75 +436,85 @@ def __hash__(self):
     ],
 )
 
```

Comment:
I will only use `pyspark_sql` as a test here. If this idea is reasonable, I will proceed with the other modules.

Reply:
cc @HyukjinKwon @dongjoon-hyun FYI

Reply:
It makes sense to skip when there are few but I think we shouldn't just skip them like this if there are too many.

Reply:
We should fix instead of just skipping them.

Reply:
Or for the daily test of  Java 21, disable PySpark test first? The failed tests also need wait for the new arrow version be upgrade.

Reply:
Yeah, we could just don't run PySpark tests for now in the daily build.

Reply:
Ok，let me to do it.

Reply:
Done